### DISCUSSION TITLE
Windows Server 2012 R2 Datacenter run CefSharp 120.2.7 error

### DISCUSSION THREAD

**Discussion by powersunmark** [cefsharp/CefSharp#4698] [Q&A]
use CefSharp  120.2.7 in win11  with VS2019, app can complie and run ok. 
i want to move  app to  Windows Server 2012 R2 Datacenter to code. 
following code show errmsg :  
System.IO.FileLoadException
  HResult=0x8007007F
  Message=未能加载由“CefSharp.Core.Runtime.dll”导入的过程。
  Source=CefSharp.Core
  StackTrace:
   at CefSharp.CefSettingsBase..ctor() ...
system has installed  vc_redist.x64.exe [Visual Studio 2015, 2017, 2019, and 2022]
--
Help me ...

<br class="Apple-interchange-newline">
</body>
</html>
Code:
var settings = new CefSettings()
{
	LogFile = "Debug.log", //You can customise this path
	LogSeverity = LogSeverity.Default // You can change the log level
};

**Comment by powersunmark** [comment]
use C# Framework 4.8

**Comment by amaitland** [comment]
> Windows Server 2012 R2 Datacenter

No longer supported (https://github.com/cefsharp/CefSharp/discussions/4336)
